lblod/frontend-lokaal-mandatenbeheer
# Remove two unused components

## Description

Removed two components that were no longer in use.

## How to test

You can see if the applications still works, but that is a bit tiresome, you could also search the application to see if these components were indeed not used anymore.
